layout/js/map.js

```diff
@@ -29,6 +29,19 @@ var animate_resize_map = function(data) {
   });
 };
 
+var animate_update_tooltips = function(data) {
+  $.each(data, function() {
+    // create the tooltip text
+    var newtip = this.state_name + ': ' + this.value + ' million gallons per day'
+    // this section doesn't work. trying to assign that text to the hovertext 
+    // call for onmousemove for each object, but haven't found the way. help!
+    var newtipobj = {
+      "onmousemove": "hovertext('" + newtip + "', evt);"
+    };
+    $("#" + this.state_name + "-mouseover").mousemove = hovertext(newtip);
```

Comment:
hmmm... I don't know enough jquery to be very useful here. But, I would suggest that tooltip text gets changed based on `mousemove`, but also that changing categories or time also wipes the text and resets it according to values that are active. This is something I haven't tackled before - with hurricane matthew, we didn't use tooltips that gave the current value of anything because of the risk of having the mouse sit in a spot and press play and the tooltip not being updated (would be misleading). I haven't played out with good ways to clear it, but this is a common pattern that would be useful to figure out. At present, we have just been avoiding it.